services/viva/microservice/serverless.yml

```diff
@@ -256,7 +258,11 @@ functions:
               - 'Completions Due Date Probably Overdue'
             detail:
               caseStatusType: [{ 'anything-but': { 'prefix': 'active:processing' } }]
-              caseState: [{ 'anything-but': 'VIVA_APPLICATION_RECEIVED' }]
+              caseState:
+                [
```

Comment:
Because of the rules this lambda seems to run always. [Docs](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns.html) says something about the value being an array:

```yaml
{ 'anything-but': ['VIVA_APPLICATION_RECEIVED', 'VIVA_APPLICATION_LOCKED'] }
```

But doesn't fully explain what that means..